src/applications/disability-benefits/all-claims/tests/automated-tests.puppeteer.spec.js

```diff
@@ -32,8 +33,26 @@ const testConfig = {
       await page.waitFor('.usa-button-primary:not(.schemaform-start-button)');
       await page.click('.usa-button-primary');
     },
-    // TODO: Add a hook for the rated disabilities page
-    // TODO: Add a hook for the bank info page
+    '/disability/file-disability-claim-form-21-526ez/disabilities/rated-disabilities': async page => {
+      await page.click('input[name="root_ratedDisabilities_0"]');
+      await page.click('.form-progress-buttons .usa-button-primary');
+    },
+    '/disability/file-disability-claim-form-21-526ez/payment-information': async (
+      page,
+      data,
+      config,
+      log,
+    ) => {
+      if (data['view:bankAccount']) {
+        if (await page.$('.usa-button-primary.edit-button')) {
```

Comment:
Ooh, clever. I didn't think about just clicking the edit button if it existed. I like it! :raised_hands: